coalib/settings/ConfigurationGathering.py

```diff
@@ -4,7 +4,7 @@
 import logging
 
 from coalib.collecting.Collectors import (
-    collect_all_bears_from_sections, filter_section_bears_by_languages)
+                                collect_all_bears_from_sections, filter_section_bears_by_languages)  # Ignore PEP8Bear
```

Comment:
Unrelated change.

Reply:
Split into a separate commit if you still want to add this change in.